.github/pythonpackage.yml

```diff
@@ -15,12 +15,12 @@ jobs:
         python-version: '3.x'
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        python3 -m pip install --upgrade pip
         pip install setuptools wheel twine
```

Comment:
not sure if this should be `pip3` to clarify which pip is used.
Had some troubles since pip may point to a python2 interpreter, instead of python3.

Reply:
We are explicitly asking the python3 to load `pip` module which is much safer than using `pip3` as some of the machines will have it only as `pip`

Reply:
please keep it like "python3 -m pip ...."